### PR TITLE
feat(TCOMP-547):update rest start script to use the PAX_MVN_REPO env var

### PR DESCRIPTION
**Pull request by sgandon**
**What is the current behavior?** (You can also link to an open issue here)
Web Service cannot be customized to get components from remote repo (nor the docker image)

**What is the new behavior?**
Service and docker image accepts a new environment variable PAX_MVN_REPO to configure repo repo used to look for components and dependencies.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/322/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/322/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/326/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/326/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/340/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/340/)

File: services/components-api-service-rest/scripts/start.sh [hunk 1]
```diff
@@ -62,6 +62,16 @@ if [ ! -z "$KRB5_CONFIG" ] ; then
   JAVA_OPTS="$JAVA_OPTS -Djava.security.krb5.conf=$KRB5_CONFIG"
 fi
 
+
```
Comment (gonchar-ivan) [inline services/components-api-service-rest/scripts/start.sh]: What about adding this option for start.bat too?

Comment (sgandon) [inline services/components-api-service-rest/scripts/start.sh]: indeed, I'll add this to https://jira.talendforge.org/browse/TCOMP-553

**Pull request merged**